molecule/source-static/host_vars/debian-11

```diff
@@ -14,6 +14,7 @@ pulp_install_plugins:
   #   source_dir: "/var/lib/pulp/devel/pulp_deb"
   pulp-file:
     git_url: "https://github.com/pulp/pulp_file"
+    git_revision: "main"
```

Comment:
Would not specifying it default to the primary branch?

Reply:
If you just want to keep a record of this functionality, we have an example here:

https://github.com/pulp/pulp_installer/blob/master/example.dev-config.yml#L13

Reply:
it breaks
https://github.com/pulp/pulp_installer/runs/3286411123?check_suite_focus=true#step:7:5670
it seems the issue is here
https://github.com/pulp/pulp_installer/blob/master/roles/pulp_common/templates/requirements.in.j2#L4

Reply:
I see. We should stop calling it optional in the example then.
Also i'm wondering if you could specify the git+url in a way that does not include the revision. Maybe you'd need to cut out the "@" too.